config/nav.yml

```diff
@@ -53,6 +53,7 @@ nav:
         # Serving config
         - Knative Serving configuration:
           - Configure Deployment resources: admin/serving/deployment.md
+          - Configuring gradual roll-out of traffic to Revisions: admin/serving/rolling-out-latest-revision-configmap.md
```

Comment:
```suggestion
          - Configuring gradual rollout of traffic to Revisions: admin/serving/rolling-out-latest-revision-configmap.md
```

Reply:
[Merriam-Webster spells rollout without a hyphen](https://www.merriam-webster.com/dictionary/rollout)
[Ngrams Viewer shows it's the popular preference](https://books.google.com/ngrams/graph?content=roll-out%2Crollout&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Croll%20-%20out%3B%2Cc0%3B.t1%3B%2Crollout%3B%2Cc0)